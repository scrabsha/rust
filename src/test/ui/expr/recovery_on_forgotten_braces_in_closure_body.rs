// This test corresponds to issue #88065 which points out that there is no
// recovery when parsing a block closure body when there is no braces around
// its statements.

fn main() {
    let num = 5;
    (1..num).reduce(|a, b| 
        println!("{}", a); //~ ERROR: Missing braces around block
        a * b
    ).unwrap();
}
