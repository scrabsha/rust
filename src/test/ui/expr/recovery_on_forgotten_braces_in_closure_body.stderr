error: Missing braces around block
  --> $DIR/recovery_on_forgotten_braces_in_closure_body.rs:8:9
   |
LL | /         println!("{}", a);
LL | |         a * b
   | |_____________^

error: aborting due to previous error

